### PR TITLE
APPSRE-10351 print state url

### PR DESCRIPTION
**Pull request by fishi0x01**
Print quick access link to the saas deploy state file in S3 for further debugging.

### CODE REVIEW

File: tools/qontract_cli.py [hunk 1]
```diff
@@ -3680,14 +3680,18 @@ def get_promotion_state(channel: str, sha: str):
...
     for publisher_id, state in promotion_state.get(channel=channel, sha=sha).items():
         print()
         if not state:
             print(f"No state found for {publisher_id=}")
         else:
-            print(f"State for {publisher_id=}:")
-            print(state)
+            print(f"{publisher_id=}")
+            print(
+                f"State link: https://us-east-1.console.aws.amazon.com/s3/object/{bucket}?region=us-east-1&bucketType=general&prefix=state/openshift-saas-deploy/promotions_v2/{channel}/{publisher_id}/{sha}"
```
Comment (hemslo) [inline tools/qontract_cli.py:3692]: Can we get region from env var? `us-east-1` can be default if not provided

**Review by hemslo — APPROVED** [review]
LGTM!

File: tools/qontract_cli.py (hunk 1 above)
Comment (fishi0x01) [reply tools/qontract_cli.py]: Good idea. Added from env var

Comment (hemslo) [reply tools/qontract_cli.py]: @fishi0x01 missed one replace in string `?region=us-east-1`

Comment (fishi0x01) [reply tools/qontract_cli.py]: Good catch - adding here https://github.com/app-sre/qontract-reconcile/pull/4455

**Pull request merged**